packages/desktop-client/src/components/payees/PayeeTableRow.tsx

```diff
@@ -46,11 +47,11 @@ function RuleButton({ ruleCount, focused, onEdit, onClick }: RuleButtonProps) {
       >
         <Text style={{ paddingRight: 5 }}>
           {ruleCount > 0 ? (
-            <>
+            <Trans>
               {ruleCount} associated {ruleCount === 1 ? 'rule' : 'rules'}
```

Comment:
This looks to be leading to an error:

![image](https://github.com/user-attachments/assets/3db95dac-907c-48d0-bb92-26f30b2fde23)